em_examples/Base.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, widgets, kwargs):
 
     @property
     def kwargs(self):
-        instanceCheck = lambda x: isinstance(x, ToggleButtons) or isinstance(x, FloatSlider) or isinstance(x, IntSlider) or isinstance(x, FloatText) or isinstance(x, fixed) or isinstance(x, Checkbox)
+        instanceCheck = lambda x: isinstance(x, ToggleButton) or isinstance(x, ToggleButtons) or isinstance(x, FloatSlider) or isinstance(x, IntSlider) or isinstance(x, FloatText) or isinstance(x, fixed) or isinstance(x, Checkbox)
```

Comment:
@thast, could you instead just check if it is a ipywidget.Widget? 
![image](https://cloud.githubusercontent.com/assets/6361812/26656444/e8e3314c-4613-11e7-8d31-c24021e00e6c.png)

FloadSliders, Toggles, etc should inherit the Widget, so it should be sufficient to check that